NextcloudTalk/PollCreationViewController.swift

```diff
@@ -192,41 +192,41 @@ import UIKit
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let textInputCell = tableView.dequeueReusableCell(withIdentifier: kTextInputCellIdentifier) as? TextInputTableViewCell ??
-        TextInputTableViewCell(style: .default, reuseIdentifier: kTextInputCellIdentifier)
-        textInputCell.textField.delegate = self
-        textInputCell.textField.autocapitalizationType = .sentences
-        let actionCell = tableView.dequeueReusableCell(withIdentifier: "PollSettingCellIdentifier") ?? UITableViewCell(style: .default, reuseIdentifier: "PollSettingCellIdentifier")
-
         if indexPath.section == PollCreationSection.kPollCreationSectionQuestion.rawValue {
+            let textInputCell: TextFieldTableViewCell = tableView.dequeueOrCreateCell(withIdentifier: textFieldCellIdentifier)
+            textInputCell.textField.delegate = self
+            textInputCell.textField.autocapitalizationType = .sentences
             textInputCell.textField.placeholder = NSLocalizedString("Ask a question", comment: "")
             textInputCell.textField.tag = kQuestionTextFieldTag
             textInputCell.textField.text = question
             return textInputCell
         } else if indexPath.section == PollCreationSection.kPollCreationSectionOptions.rawValue {
             if indexPath.row == options.count {
+                let actionCell = tableView.dequeueOrCreateCell(withIdentifier: "PollSettingCellIdentifier")
                 actionCell.textLabel?.text = NSLocalizedString("Add answer", comment: "")
                 return actionCell
             } else if indexPath.row < options.count {
+                let textInputCell: TextFieldTableViewCell = tableView.dequeueOrCreateCell(withIdentifier: textFieldCellIdentifier)
```

Comment:
The delegate is missing here, resulting in non-trimmed answeres. Also autocapitalizationType is missing, but since it's the default value anyway (https://developer.apple.com/documentation/uikit/uitextinputtraits/1624447-autocapitalizationtype) it doesn't make a difference.

Edit: Ah no, since you reset the type in `prepareForReuse`, we need to set it here again, as otherwise we loose the autocap. functionality. Which makes me wonder, should reuse and init at least do the same?